contracts/buy-and-make/RevenueBuyBack.sol

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.6;
+
+import { IEmissionsController } from "../interfaces/IEmissionsController.sol";
+import { IMasset } from "../interfaces/IMasset.sol";
+import { IRevenueRecipient } from "../interfaces/IRevenueRecipient.sol";
+import { ImmutableModule } from "../shared/ImmutableModule.sol";
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IUniswapV3SwapRouter } from "../peripheral/Uniswap/IUniswapV3SwapRouter.sol";
+
+struct RevenueBuyBackConfig {
+    // Percentage of mAssets that can be redeemed for bAssets in basis points (BASSET_PECENTAGE_SCALE).
+    // So if a 2% slippage is allowed, the minRedeemSlippage will be 9800.
+    // A 0.01% slippage is 9999
+    uint16 minBassetPercentage;
```

Comment:
May want to scale both this var and `minBasset2RewardsPrice` up to some higher amount of 0's in order to accommodate different decimals

Reply:
I've change both to use 1e18